examples/gno.land/p/demo/grc/grc721/basic_nft.gno

```diff
@@ -69,6 +69,25 @@ func (s *basicNFT) TokenURI(tid TokenID) (string, error) {
 	return uri.(string), nil
 }
 
+func (s *basicNFT) SetTokenURI(tid TokenID, tURI TokenURI) (bool, error) {
+	// check for invalid TokenID
+	if !s.exists(tid) {
+		return false, ErrInvalidTokenId
```

Comment:
let's leave it as is for this PR, but I'm going to make one changing all instances of Id -> ID...